ops/terraform/services/base/values/test.yaml

```diff
@@ -27,30 +27,33 @@
 /bfd/test/migrator/nonsensitive/db_migrator_tmp_dir: /opt/bfd-db-migrator/tmp
 
 # PIPELINE CONFIGURATION
-/bfd/${env}/pipeline/nonsensitive/bfd_pipeline_dir: &bfdPipelineDir /bluebutton-data-pipeline
-/bfd/${env}/pipeline/nonsensitive/data_pipeline_dir: *bfdPipelineDir
-/bfd/${env}/pipeline/nonsensitive/data_pipeline_new_relic_metric_host: *newRelicMetricHost
-/bfd/${env}/pipeline/nonsensitive/data_pipeline_new_relic_metric_path: *newRelicMetricPath
-/bfd/${env}/pipeline/nonsensitive/data_pipeline_user: bb-etl
-/bfd/${env}/pipeline/nonsensitive/instance_type: m5.xlarge
-/bfd/${env}/pipeline/nonsensitive/data_pipeline_micrometer_cw_enabled: false
-/bfd/${env}/pipeline/nonsensitive/data_pipeline_micrometer_cw_namespace: bfd-${env}/bfd-pipeline
-/bfd/${env}/pipeline/nonsensitive/data_pipeline_micrometer_cw_interval: PT1M
+/bfd/${env}/pipeline/shared/nonsensitive/bfd_pipeline_dir: &bfdPipelineDir /bluebutton-data-pipeline
+/bfd/${env}/pipeline/shared/nonsensitive/data_pipeline_dir: *bfdPipelineDir
+/bfd/${env}/pipeline/shared/nonsensitive/data_pipeline_new_relic_metric_host: *newRelicMetricHost
+/bfd/${env}/pipeline/shared/nonsensitive/data_pipeline_new_relic_metric_path: *newRelicMetricPath
+/bfd/${env}/pipeline/shared/nonsensitive/data_pipeline_user: bb-etl
+/bfd/${env}/pipeline/shared/nonsensitive/data_pipeline_micrometer_cw_enabled: false
+/bfd/${env}/pipeline/shared/nonsensitive/data_pipeline_micrometer_cw_namespace: bfd-${env}/bfd-pipeline
```

Comment:
Only commenting on the `test` variant for brevity, and _really_ only commenting at all for posterity. 

Discussed this with @brianburton and we believe this *should* work without further modification but we _might_ want to consider separating the CloudWatch namespaces as we begin to record other metrics in CloudWatch via Micrometer. Then again, we _might_ not want to do that at all. Let's leave it be, and iterate.